R/pkg/R/functions.R

```diff
@@ -2254,40 +2255,48 @@ setMethod("date_format", signature(y = "Column", x = "character"),
             column(jc)
           })
 
+setClassUnion("characterOrstructTypeOrColumn", c("character", "structType", "Column"))
```

Comment:
we should probably try to pull all the setClassUnion in one place. (to avoid conflict or duplication)

Reply:
Yup, I agree.. Would you mind if I do this separately? I roughly checked by `grep` and looks:

```
./pkg/R/DataFrame.R:setClassUnion("characterOrstructType", c("character", "structType"))
./pkg/R/DataFrame.R:setClassUnion("numericOrcharacter", c("numeric", "character"))
./pkg/R/DataFrame.R:setClassUnion("characterOrColumn", c("character", "Column"))
./pkg/R/DataFrame.R:setClassUnion("numericOrColumn", c("numeric", "Column"))
```

Reply:
yes